src/redux/config/state.ts

```diff
@@ -0,0 +1,15 @@
+export type Message = any;
```

Comment:
I'm not sure what the `Message` actual type is. Feel free to amend.

Reply:
It depends on the query. In a normal scenario it would be a generic, but we can leave as is.